src/polycap-private.h

```diff
@@ -142,5 +158,82 @@ struct polycap_result
   double *absorb_sum, *sum_cnt;
   };
 
+//================================
+
+#include "hdf5.h"
+// Write data set in HDF5 file
+void polycap_h5_write_dataset(hid_t file, int rank, hsize_t *dim, char *dataset_name, double *data, char *unitname);
```

Comment:
Deze declaratie en de hdf5 include mogen hier weg, want ze worden toch enkel in `polycap-out.c` gebruikt.

Dit zal de build failure op de AppVeyor fiksen.

Reply:
By the way: nog gedacht aan een abstract voor een poster over polycap voor EXRS? Deadline is 31 maart...

Reply:
Goh, ja, daar moet ik idd ook nog eens voor kijken :P 
Zal het aan mijn to-do lijstje voegen. Momenteel nog wat aan het prutsen aan de code. Een soort van gebrekkige berekening gevonden, maar kzie niet goed in waarom het verschillende resultaten geeft in mijn/ons code dan in Laszlo's code :)